----------------------------------------------------------------------
Timestamp - 02/03/2022 09:30:48 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11060
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:31:14 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11060
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:31:43 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11060
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:32:50 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000047-0001-fd00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|85c6a685-41457cb0ea8df6f2."
TraceId : "85c6a685-41457cb0ea8df6f2"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11060
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:34:42 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:35:03 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:35:27 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:36:43 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000003a-0002-fe00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|e542df31-4e9a8669a1b6f31b."
TraceId : "e542df31-4e9a8669a1b6f31b"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:36:45 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000012-0000-fd00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|e542df32-4e9a8669a1b6f31b."
TraceId : "e542df32-4e9a8669a1b6f31b"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:38:52 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000003c-0002-fe00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|e542df33-4e9a8669a1b6f31b."
TraceId : "e542df33-4e9a8669a1b6f31b"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:42:56 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000049-0001-fd00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|e542df34-4e9a8669a1b6f31b."
TraceId : "e542df34-4e9a8669a1b6f31b"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 11752
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:45:19 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 14740
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:45:39 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 14740
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:46:02 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 14740
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:46:40 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000a-0002-fd00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|b9f55193-4e0871b745e88ca4."
TraceId : "b9f55193-4e0871b745e88ca4"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 14740
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:49:50 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17140
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:50:12 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17140
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:50:40 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17140
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:51:11 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000c-0002-fd00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|bf8763c5-4d49676c1ed0fcd9."
TraceId : "bf8763c5-4d49676c1ed0fcd9"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17140
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:59:06 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 18404
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 09:59:50 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 18404
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:00:24 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 18404
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:00:32 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000e-0002-fd00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|2a3f93a4-40f0f6081bb3ff5e."
TraceId : "2a3f93a4-40f0f6081bb3ff5e"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 18404
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:04:48 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17532
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:05:16 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17532
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:05:46 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17532
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:05:47 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 17532
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:14:24 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10752
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:14:53 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10752
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:15:17 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10752
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:15:49 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000006-0000-fb00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|747e60ae-4c99bf065a047020."
TraceId : "747e60ae-4c99bf065a047020"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10752
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:17:21 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000008-0001-fb00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|747e60af-4c99bf065a047020."
TraceId : "747e60af-4c99bf065a047020"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10752
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:20:38 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 16072
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:20:59 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 16072
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:21:30 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 16072
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:21:59 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000020-0000-fe00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|b20d5c32-4a278033477a2396."
TraceId : "b20d5c32-4a278033477a2396"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 16072
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:23:36 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 5656
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:23:58 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 5656
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:24:23 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 5656
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:24:57 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000024-0000-fe00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|3040afd0-46ee31ee2c2d76ca."
TraceId : "3040afd0-46ee31ee2c2d76ca"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 5656
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 20 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:30:58 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 20568
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:31:23 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 20568
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:31:50 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 20568
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:37:11 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10180
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:37:37 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10180
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:38:02 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10180
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:42:30 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000001b-0000-fd00-b63f-84710c7967bb"
RequestPath : "/ws"
SpanId : "|f14e39ea-4939eb0bbaec668e."
TraceId : "f14e39ea-4939eb0bbaec668e"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10180
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:42:30 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000004-0000-f900-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|f14e39eb-4939eb0bbaec668e."
TraceId : "f14e39eb-4939eb0bbaec668e"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 10180
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.<Invoke>g__AwaitMatcher|8_0(EndpointRoutingMiddleware middleware, HttpContext httpContext, Task`1 matcherTask)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.<Invoke>g__AwaitMatcher|8_0(EndpointRoutingMiddleware middleware, HttpContext httpContext, Task`1 matcherTask)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:47:12 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 1860
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:47:34 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 1860
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:47:59 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 1860
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:52:35 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15988
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:52:56 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15988
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:56:45 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15988
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 10:57:37 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "8000000f-0000-fb00-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|d2a60ed1-4945b3ad0e4ccbea."
TraceId : "d2a60ed1-4945b3ad0e4ccbea"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 15988
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:02:50 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12472
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:03:28 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12472
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:03:48 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12472
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:08:44 +01:00 | Level - Error |
----------------------------------------------------------------------
EventId : { Id: 1, Name: "UnhandledException" }
SourceContext : "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware"
RequestId : "80000005-0001-f900-b63f-84710c7967bb"
RequestPath : "/"
SpanId : "|55bf0133-4f69e8e3272fee35."
TraceId : "55bf0133-4f69e8e3272fee35"
ParentId : ""
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 12472
ProcessName : "iisexpress"
----------------------EXCEPTION DETAILS-------------------------------
Exception - System.TimeoutException: The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.
   at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)StackTrace -    at Microsoft.AspNetCore.SpaServices.Extensions.Util.TaskTimeoutExtensions.WithTimeout[T](Task`1 task, TimeSpan timeoutDelay, String message)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)Message - The Angular CLI process did not start listening for requests within the timeout period of 0 seconds. Check the log output for error information.Source - Microsoft.AspNetCore.SpaServices.ExtensionsInnerException - ---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:22:26 +01:00 | Level - Error |
----------------------------------------------------------------------
Error : "One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.))"
StackTrace : "   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at System.Threading.Tasks.Task.Wait()
   at CMS_CORE_NG.Program.Main(String[] args) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\CMS_CORE_NG\Program.cs:line 46"
InnerException : "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.)
 ---> System.ComponentModel.Win32Exception (10061): Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée.
   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabaseCreator.EnsureCreatedAsync(CancellationToken cancellationToken)
   at DataService.DbContextInitializer.Initialize(DataProtectionKeysContext dataProtectionKeysContext, ApplicationDbContext applicationDbContext, IFunctionalSvc functionalSvc) in C:\Users\sabrine\Desktop\CMS_CORE_NG\CMS_CORE_NG\DataService\DbContextInitializer.cs:line 16
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:10061,State:0,Class:20"
Source : "System.Private.CoreLib"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9272
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:22:47 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9272
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:23:15 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9272
ProcessName : "iisexpress"
---------------------------------------------------------------------------
----------------------------------------------------------------------
Timestamp - 02/03/2022 11:23:17 +01:00 | Level - Error |
----------------------------------------------------------------------
SourceContext : "Microsoft.AspNetCore.SpaServices"
Application : "CMS_CORE_NG"
MachineName : "DESKTOP-O24T72T"
CurrentManagedThreadId : 1
OSVersion : "Microsoft Windows NT 10.0.19043.0"
Version : "3.1.20"
UserName : "sabrine"
ProcessId : 9272
ProcessName : "iisexpress"
---------------------------------------------------------------------------
